service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/StandardDirectionOrderIssuedEmailContentProvider.java

```diff
@@ -35,4 +53,56 @@ public AllocatedJudgeTemplateForSDO buildNotificationParametersForAllocatedJudge
 
         return allocatedJudgeTemplate;
     }
+
+    public CTSCTemplateForSDO buildNotificationParametersForCTSC(CaseDetails caseDetails) {
+        CaseData caseData = mapper.convertValue(caseDetails.getData(), CaseData.class);
+
+        HearingBooking hearing = caseData.getFirstHearing().orElseThrow(NoHearingBookingException::new);
```

Comment:
Do we want to put some sort of exception message here?

Reply:
the exception returns the message: "Expected to have at least one hearing booking"

Reply:
Ah yes because it is a custom exception 👍